src/tracer.js

```diff
@@ -74,6 +74,7 @@ export default class Tracer {
             spanContext: SpanContext,
             operationName: string,
             startTime: number,
+            internalTags: any = [],
```

Comment:
we're doing a lot of defaulting by creating new objects, I wonder how that impacts the benchmarks.